### DISCUSSION TITLE
ChanServ Mute Quiet setting

### DISCUSSION THREAD

**Discussion by legendskyk** [anope/anope#453] [Support] [answered]
ChanServ not support mute or quiet command how it possible to make it?  also alias command like if room admins ops type !m nick then user got muted by service bot

**Comment by sadiepowell** [comment, marked as answer]
The default configs include a !mute command. In order to use fantasy commands you need either the FANTASIA (2.0) or FANTASY (2.1) privilege and the fantasy setting needs to be on for the channel.

https://github.com/anope/anope/blob/2.0/data/botserv.example.conf#L389

https://github.com/anope/anope/blob/2.0/data/botserv.example.conf#L352

**Comment by legendskyk** [comment]
Its already configured in my anope but how it worked?

**Reply by sadiepowell** [reply]
If you have those settings on you can just do !mute <nick>.
